onnxruntime/core/providers/cuda/object_detection/non_max_suppression.cc

```diff
@@ -94,40 +94,35 @@ Status NonMaxSuppression::ComputeInternal(OpKernelContext* ctx) const {
     Tensor* output = ctx->Output(0, {static_cast<int64_t>(total_num_saved_outputs), last_dim});
     ORT_ENFORCE(output != nullptr);
     int64_t* dst = output->MutableData<int64_t>();
-    size_t count = all_selected_indices.size();
+    int32_t count = static_cast<int32_t>(all_selected_indices.size());
```

Comment:
>all_selected_indices [](start = 41, length = 20)

not sure if the default array size of 8 would be enough. This value depends on selected bounding box, and may pass test but crash in real scene.